### DISCUSSION TITLE
Welcome to Open Decision Framework Discussions!

### DISCUSSION THREAD

**Discussion by jimmysjolund** [opensourceway/open-decision-framework#68] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jimmysjolund** [comment]
I am Jimmy Sjölund and one of the maintainers for the community version of the Open Decision Framework. I have been an Open Organization Ambassador since 2017. While I was not part of the creation of the ODF I have been using it in my time working at Red Hat. As the ODF and community kind of was forgotten for a while I worked internally to move the community version of the ODF to the Open Organization project and community. It will help to update the framework, get the community involved again, and also many of the Ambassadors was part of creating the ODF back in the day. It was a natural step.

The past years, I have been talking about the ODF in different conferences:

- [Agila Sverige 2023](https://youtu.be/IMme_ZNEj1g?si=GpVxf9AFWVB9N2cE)
- [foss-north 2024](https://youtu.be/4gMzW7LpA7s?si=qIZKaE5_lAXKimTj)
- [DevConf.cz 2024](https://youtu.be/tgsgm97p1lc?si=PAqhORyOA7spgxuN)

**Comment by semioticrobotic** [comment]
Very glad to see this happening! Thanks, @jimmysjolund, for taking it on and coordinating this much-welcome new release of the framework.
